### ISSUE TITLE
A way to remove all theme and css style of tables

### ISSUE THREAD

**Issue by dongweigogo** [hughjonesd/huxtable#241]
I am currently using huxtable for displaying some table and it is working good. 
But I found that the css style is always added to huxtable object even when I use theme_basic(). Although the themes in huxtable are nice, I prefer my own ones. 
Thus, I would like to know if there is a way to remove the buildin themes and css style. The raw HTML code is probably what I need.

**Comment by hughjonesd** [comment]
Hmm, seems reasonable. I'll look into it.

**Comment by hughjonesd** [comment]
A workaround right now might be to capture the HTML with to_html and then edit it with tools from another package.

**Comment by hughjonesd** [comment]
Maybe an argument to `to_html` with a default `options(huxtable.strip_css)`would make sense, because probably people will want to do this for many tables in a document.

**Comment by hughjonesd** [comment]
Hmm. Let me check. Do you really mean all the CSS - heights, colours, everything?

If so, that will remove most of the functionality. So then I think the best thing would be a new function `to_basic_html`. But I might also think that this is beyond the package scope, and you'd do better to strip the CSS using a different tool.

**Comment by dongweigogo** [comment]
> t.
I agree that CSS matters a lot with functionality, actually the buildin css style changes the padding, background color, border width in my case. Any approach to that is OK with me. Thanks.

**Comment by hughjonesd** [comment]
So, I think the best way to do this is with xml2. Here is an example:

```r
library(huxtable)
library(xml2)
jams
jams_tbl <- jams |> 
  huxtable::as_html() |>
  read_html() |>
  xml_find_first("./table")

xml_set_attr(jams_tbl, "class", "myclass") # modification in place, unusual for R 
tds <- xml_find_all(jams_tbl, ".//td")
xml_set_attr(tds, "style", "") # remove all styles

as.character(jams_tbl)
```

I'm still open to doing this if there is demand. There may also be better HTML manipulation libraries that I don't know about.
